web/app.fluidity.money/app/routes/$network/dashboard/home.tsx

```diff
@@ -405,9 +407,15 @@ export default function Home() {
 
     const shouldMultiplyBottles = application != "none";
 
-      let adjustedLootboxCount = lootboxCount;
-      if (shouldMultiplyBottles)
-        adjustedLootboxCount = adjustedLootboxCount * ADJUSTED_BOTTLE_MULTIPLIER;
+    // reward lifi bottles disproporationately (24x) compared to the usual (12x)
+
+    let adjustedLootboxCount = lootboxCount;
+    switch (true) {
```

Comment:
this has redundant logic (if application is lifi shouldMultiplyBottles is always going to be true), and should be a ternary such as 
```typescript 
const adjustedLootboxCount = application === "lifi"
  ? lootboxCount * lifiMultiplier 
  : application !== "none" 
    ? lootboxCount * bottleMultiplier 
    : lootboxCount
```